prow/config.yaml

```diff
@@ -19926,13 +19923,7 @@ periodics:
       - name: ssh
         mountPath: /etc/ssh-key-secret
         readOnly: true
-      - name: docker-graph
-        mountPath: /docker-graph
-      # TODO(bentheelder): sort out container port
-      ports:
-      - containerPort: 9999
-        hostPort: 9999
-      # docker-in-docker needs privilged mode
+      # bazel needs privilged mode
```

Comment:
nit: `bazel needs privileged mode to sandbox builds` is more precise and fixes a typo :-)